src/win/snprintf.c

```diff
@@ -0,0 +1,49 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
Meta: I think we should get rid of this copyright boilerplate.  Joyent hasn't held any copyright to new libuv code for a long time.